### ISSUE TITLE
Make setup.py clean remove in place build files in pixell directory

### ISSUE THREAD

**Issue by msyriac** [simonsobs/pixell#33]

**Comment by mhasself** [comment]
I don't think I should be assigned to this... I don't really like the "develop" mode.  Modern build systems simplify bookkeeping by keeping built/installable files totally separate from the source files.  "setup.py develop" is a hack around this nice practice.

**Comment by msyriac** [comment]
Ok, I will take care of this. Could you clarify what your preferred way of working in "development mode" is? e.g. do you re-install the package every time you make a change?

**Comment by mhasself** [comment]
Yes, I reinstall every time.  I often write my own Makefile that wraps the commands I need; e.g. "make install_user" might invoke something more verbose, such as
  python3 setup.py install --user
If I can't spare a second terminal, I'll often use emacs' "compile" command, to invoke "make install_user -C .." or similar.  That makes for fast develop -> debug cycles.

**Comment by msyriac** [comment]
Better yet, remove setup.py! Closing.
